src/main/resources/view/LightTheme.css

```diff
@@ -196,6 +196,18 @@
     -fx-text-fill: #010504;
 }
 
+.info_big_label {
```

Comment:
Hi Shi Jun, is it possible to make the list panel on the left side change to a lighter colour to match the"light theme"? Otherwise good work, the theme looks aesthetic.

Reply:
will fix it in the next PR